pkg/hooks/hooks.go

```diff
@@ -110,7 +110,7 @@ type Hooks struct {
 	ShouldAddDistributedInfoHook func() bool
 
 	// Distributed query
-	InitDistributedQueryServiceHook func(querySummary interface{}, allSegFileResults interface{}, distQueryId string) interface{}
+	InitDistributedQueryServiceHook func(querySummary interface{}, allSegFileResults interface{}, distQueryId string, segEncToKey uint32) interface{}
```

Comment:
Since this is not a map, maybe rename to something like `segKeyEnc`. Or maybe you meant to pass a map?